central/node/service/service_impl.go

```diff
@@ -69,19 +65,11 @@ func (s *nodeServiceImpl) ListNodes(ctx context.Context, req *v1.ListNodesReques
 }
 
 func (s *nodeServiceImpl) GetNode(ctx context.Context, req *v1.GetNodeRequest) (*storage.Node, error) {
-	clusterLocalStore, err := s.nodeStore.GetClusterNodeStore(ctx, req.GetClusterId(), false)
-	if err != nil {
-		return nil, errors.Errorf("could not access per-cluster node store for cluster %q: %v", req.GetClusterId(), err)
-	}
-
-	node, err := clusterLocalStore.GetNode(req.GetNodeId())
-	if err != nil {
-		return nil, errors.Errorf("could not locate node %q in per-cluster node store for cluster %s: %v", req.GetNodeId(), req.GetClusterId(), err)
+	// Nodes have non-composite unique node ID, hence, cluster ID is not required to retrieve a node.
+	// Note that whether we provide clusterID or not, the searcher will always perform SAC check in the graph.
```

Comment:
Where would `clusterID` factor in this? Looks like this is a `Get` instead of a `Search`

Reply:
The way I read this comment is that the request makes it look like you need both clusterID _and_ nodeID but apparently you don't since nodeID is unique across all clusters? Is that correct?

If so, perhaps reword this to be explicit that nodeID is unique across all clusters and that clusterID in the request is not required.

Reply:
> Looks like this is a Get instead of a Search

@RTann, Searcher in `GetNode` is the implementation detail of GetNode. Reworded the comment.

> If so, perhaps reword this to be explicit that nodeID is unique across all clusters and that clusterID in the request is not required.

@theencee, to avoid any user-facing changes, I will not change the API. Although, it is clear to me that internally we do not require it. Added context to the comment.